### PR TITLE
Some improvements in autocompletion

### PR DESCRIPTION
**Pull request by jespino**
#### Summary
Adding the icon, and fixing some of the autocompletion for names of instances. Add autocompletion for the flags.

```release-note
None
```

### CODE REVIEW

**Comment by jespino** [comment]
This time I also tested it locally (I was able to test it with a ton of fake configuration, because I'm only testing the autocompletion)

**Review by nickmisasi — APPROVED** [review]

File: server/command.go [hunk 1]
```diff
@@ -236,6 +322,19 @@ func (p *Plugin) getCommand() *model.Command {
...
+							Type: model.AutocompleteArgTypeStaticList,
+							Data: &model.AutocompleteStaticListArg{
+								PossibleArguments: []model.AutocompleteListItem{
+									{
+										Item:     "miniSingleton",
+										HelpText: "Mini Singleton instance",
+									},
+									{
+										Item:     "miniHA",
+										HelpText: "Mini Singleton cluster",
```
Comment (gabrieljackson) [inline server/command.go:334]:
```suggestion
										HelpText: "Mini cluster made of two servers",
```

File: server/command.go [hunk 2]
```diff
@@ -167,9 +219,18 @@ func (p *Plugin) getCommand() *model.Command {
...
```
Comment (gabrieljackson) [inline server/command.go:231]:
```suggestion
										HelpText: "Mini cluster made of two servers",
```

File: server/command.go [hunk 3]
```diff
@@ -149,9 +188,22 @@ func (p *Plugin) getCommand() *model.Command {
...
 						{
-							Type: model.AutocompleteArgTypeText,
-							Data: &model.AutocompleteTextArg{
-								Hint: "filestore",
+							Type: model.AutocompleteArgTypeStaticList,
+							Data: &model.AutocompleteStaticListArg{
+								PossibleArguments: []model.AutocompleteListItem{
+									{
+										Item:     "bifrost",
+										HelpText: "RDS Postgres with pgbouncer proxy connections",
```
Comment (gabrieljackson) [inline server/command.go:196]:
```suggestion
										HelpText: "Shared S3 bucket with access via bifrost",
```

**Review by gabrieljackson — CHANGES_REQUESTED** [review]
Looks great, just found a few help texts that need a small tweak.

**Review by gabrieljackson — APPROVED** [review]
Looks great! Thanks!

**Pull request merged**